layouts/partials/icons/servicemesher.svg

```diff
@@ -0,0 +1,14 @@
+<svg viewBox="0 0 310 310">

```

Comment:
You only need this SVG if we put an icon in the page footer, and I don't think we should. So you can delete this file.